scripts/reset-test-db.sh

```diff
@@ -24,18 +24,25 @@
 
 DB_CACHE_DIR="common/test/db_cache"
 
+declare -A databases
+databases=(["default"]="edxtest" ["student_module_history"]="student_module_history_test")
+
 # Ensure the test database exists.
-echo "CREATE DATABASE IF NOT EXISTS edxtest;" | mysql -u root
+for $db in "${!databases[@]}"; do
+    echo "CREATE DATABASE IF NOT EXISTS ${databases[$db]};" | mysql -u root
 
-# Clear out the test database
-./manage.py lms --settings bok_choy reset_db --traceback --noinput
+    # Clear out the test database
+    ./manage.py lms --settings bok_choy reset_db --traceback --noinput --router $db
+done
 
 # If there are cached database schemas/data, load them
-if [[ -f $DB_CACHE_DIR/bok_choy_schema.sql && -f $DB_CACHE_DIR/bok_choy_data.json ]]; then
+if [[ -f $DB_CACHE_DIR/bok_choy_schema_$db.sql && -f $DB_CACHE_DIR/bok_choy_data_$db.json ]]; then
```

Comment:
Oh, yeah. Shoot. I don't want it to delete the `$DB_CACHE_DIR`, which is why I moved this out of the for loop, but it appears that's not going to work.